From cd2a798479a34481fafabfb5997658b51539d465 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Mon, 9 Dec 2024 11:29:24 -0700
Subject: [PATCH] Remove OS assumptions about libtoolize

* SISCone assumes that on macOS libtoolize must be glibtoolize, which
  is not the case with conda-forge libtool.
---
 autogen.sh | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 571c0a1..d8c0664 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -10,28 +10,6 @@ DIE=0
 LIBTOOL=libtool
 LIBTOOLIZE=libtoolize
 
-# change executable names when running on Macs
-# if test x`uname` = xDarwin; then
-if [ `uname` == "Darwin" ]; then
-  echo "Detected Mac OSX"
-  if [ x$(which glibtool) == "x" ]; then    
-    echo ""
-    echo "On Mac OSX, $0 requires the GNU libtool and libtoolize scripts,"
-    echo "renamed glibtol and glibtoolize respectively by Apple to avoid"
-    echo "conflict with Apple-provided libtool."
-    echo ""
-    echo "They should be already installed, together with Apple developer tools,"
-    echo "in /usr/bin. If you see this message, they are not."
-    echo ""
-    echo "The GNU version can be retrieved from http://ftp.gnu.org/gnu/libtool/."
-    echo "If they are (re)installed from sources in a different location,"
-    echo "autogen.sh should then be modified accordingly."
-    exit
-  fi
-  LIBTOOL=glibtool
-  LIBTOOLIZE=glibtoolize
-fi
-
 (autoconf --version) < /dev/null > /dev/null 2>&1 || {
   echo
   echo "**Error**: You must have \`autoconf' installed to."
-- 
2.47.1

